drools-ansible-rulebook-integration-api/src/main/java/org/drools/ansible/rulebook/integration/api/rulesengine/RegisterOnlyAgendaFilter.java

```diff
@@ -43,8 +43,8 @@ public boolean accept(Match match) {
         boolean validMatch = isValidMatch(fhs);
 
         if (validMatch) {
-            if (log.isInfoEnabled()) {
-                log.info(matchToString(match));
+            if (log.isDebugEnabled()) {
+                log.debug(matchToString(match));
```

Comment:
Requested by Madhu, changed to DEBUG level. When there are many large events, INFO would result in too much logging.